vosekast_control/LevelSensor.py

```diff
@@ -50,3 +51,10 @@ def clear_callbacks(self):
         self._gpio_controller.add_event_detect(
             self._pin, self._gpio_controller.BOTH, bouncetime=500
         )
+    
+    @property
+    def state(self):
+        return self.position
```

Comment:
The position means not the open or close state. It means the position on the tank.

Reply:
what do you propose?

Reply:
a simple solution is, to return the current state of the GPIO

Reply:
done